lib/cli/src/frameworks/common/0.Introduction.stories.mdx

```diff
@@ -0,0 +1,194 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+import Code from './assets/code-brackets.svg';
+import Colors from './assets/colors.svg';
+import Comments from './assets/comments.svg';
+import Direction from './assets/direction.svg';
+import Flow from './assets/flow.svg';
+import Plugin from './assets/plugin.svg';
+import Repo from './assets/repo.svg';
+import StackAlt from './assets/stackalt.svg';
+
+<Meta title="Example/Introduction" />
```

Comment:
Should we call this Example? Or maybe Introduction/Getting Started? I assume people could either keep this mdx because it's helpful or change it to their taste, therefore having Example wouldn't be as nice I would say

Reply:
The `Example/...` path is the future home for the rest of the example components.